### PR TITLE
update screenshot and 1line for llm obs config doc

### PR DESCRIPTION
**Pull request by barieom**
### Merge instructions

Merge readiness:
- [x] Ready for merge

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/llm_observability/configuration/_index.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Connect your OpenAI account to LLM Observability with your OpenAI API key.
 1. Select **Connect** on the OpenAI tile.
 1. Follow the instructions on the tile. 
    - Provide your OpenAI API key. Ensure that this key has **write** permission for **model capabilities**.
-1. Enable **Invoke Model from this API key**.
+1. Enable **Use this API key to evaluate your  LLM applications**.
```
Comment (aliciascott) [inline content/en/llm_observability/configuration/_index.md:34]:
```suggestion
1. Enable **Use this API key to evaluate your LLM applications**.
```

**Review by aliciascott — CHANGES_REQUESTED** [review]
Hi @barieom just one small spacing thing and then can you also keep the original image as it takes some time for the translated pages to update and it could show a broken image there. You'd just want to rename the new one like openai-tile_2.png and update that in the `img src`.

**Comment by barieom** [comment]
> Hi @barieom just one small spacing thing and then can you also keep the original image as it takes some time for the translated pages to update and it could show a broken image there. You'd just want to rename the new one like openai-tile_2.png and update that in the `img src`.

Thanks @aliciascott - addressed your comments

**Review by aliciascott — APPROVED** [review]
looks good thank you!

**Pull request merged**